### PR TITLE
feat(location): implement locate me button

### PR DESCRIPTION
**Pull request by KatvonRivia**

### CODE REVIEW

File: src/scripts/components/main/globe-navigation/globe-navigation.tsx [hunk 1]
```diff
@@ -41,8 +45,41 @@ const GlobeNavigation: FunctionComponent<Props> = ({
...
   return (
     <div className={styles.globeNavigation}>
+      {locationLoading ? (
+        <Oval className={styles.locateMe} />
+      ) : (
+        <Button
+          icon={LocationIcon}
+          className={styles.locateMe}
+          id="locate-me"
+          onClick={() => onLocateMeHandler()}
```
Comment (sadrda) [inline src/scripts/components/main/globe-navigation/globe-navigation.tsx:80]: can use onClick={onLocateMeHandler} instead

File: src/scripts/components/main/globe-navigation/globe-navigation.tsx [hunk 2]
```diff
@@ -15,6 +17,7 @@ import {GlobeProjection} from '../../../types/globe-projection';
 import {LayerListItem} from '../../../types/layer-list';
 
 import styles from './globe-navigation.module.styl';
+import {RenderMode} from '@ubilabs/esa-webgl-globe';
```
Comment (sadrda) [inline src/scripts/components/main/globe-navigation/globe-navigation.tsx:20]: get this up to the external imports

File: src/scripts/components/main/globe-navigation/globe-navigation.tsx [hunk 3]
```diff
@@ -41,8 +45,41 @@ const GlobeNavigation: FunctionComponent<Props> = ({
...
+            zoom: 0
+          };
+          dispatch(setFlyToAction(newView));
+          setLocationLoading(false);
+        },
+        error => {
+          console.error(`Error Code = ${error.code} - ${error.message}`);
+        }
+      );
+    }
```
Comment (sadrda) [inline src/scripts/components/main/globe-navigation/globe-navigation.tsx:68]: avoid nesting with `if(!navigator.geolocation) { return }`

File: src/scripts/components/main/globe-navigation/globe-navigation.tsx [hunk 4]
```diff
@@ -41,13 +45,48 @@ const GlobeNavigation: FunctionComponent<Props> = ({
...
+    setLocationLoading(true);
+
+    if (!navigator.geolocation) {
+      return;
+    }
+
+    navigator.geolocation.getCurrentPosition(
+      position => {
+        const newView = {
+          renderMode: 'globe' as RenderMode.GLOBE,
```
Comment (sadrda) [inline src/scripts/components/main/globe-navigation/globe-navigation.tsx:58]: cant you use `renderMode: RenderMode.GLOBE` here?

**Review by sadrda — APPROVED** [review]
g2g

File: src/scripts/components/main/globe-navigation/globe-navigation.tsx (hunk 4 above)
Comment (KatvonRivia) [reply src/scripts/components/main/globe-navigation/globe-navigation.tsx:58]: nope :(

**Pull request merged**